response_operations_ui/templates/collection_exercise/ci-versions.html

```diff
@@ -1,5 +1,18 @@
 {% extends "layouts/base.html" %}
+{% from "components/button/_macro.njk" import onsButton %}
 
 {% block main %}
 <h1>Choose CIR version for EQ formtype {{ form_type }}</h1>
-{% endblock %}
+<form action="" class="ons-form" method="post">
+    <input type="hidden" name="csrf_token" value="{{ csrf_token() }}" />
+    <div class="ons-grid__col">
```

Comment:
I don't think this is enough for the design system since this is a standalone html it needs a wrapping grid class and the save button should to be in line with the header as in the design